telemetry/metrics.go

```diff
@@ -90,6 +90,12 @@ type Config struct {
 	// DatadogHostname defines the hostname to use when emitting metrics to
 	// Datadog. Only utilized if MetricsSink is set to "dogstatsd".
 	DatadogHostname string `mapstructure:"datadog-hostname"`
+
+	// Otlp Exporter fields
+	OtlpExporterEnabled   bool   `mapstructure:"otlp-exporter-enabled"`
+	OtlpCollectorGrpcAddr string `mapstructure:"otlp-collector-grpc-addr"`
+	PrometheusEndpoint    string `mapstructure:"prometheus-endpoint"`
```

Comment:
It feels like this will need to be automatically populated based on what the configured prometheus metrics endpoint is?

Reply:
Yeah, each chain should probably set their own defaults.